jccli/cli.py

```diff
@@ -41,42 +41,34 @@ def cli(ctx, key):
     }
 
 @cli.command()
-@click.option('--json', "-j", required=False, type=str,
-              help='SystemUser properties json')
-@click.option('--file', "-f", required=False, type=click.Path(exists=True),
-              help='SystemUser properties file')
+@click.option('--username', '-u', required=True, type=str)
+@click.option('--email', '-e', required=True, type=str)
+@click.option('--first-name', '-f', type=str, default='')
+@click.option('--last-name', '-l', type=str, default='')
+@click.option('--allow-public-key/--disallow-public-key', default=True)
+@click.option('--ldap-binding-user', is_flag=True)
+@click.option('--passwordless-sudo', is_flag=True)
+@click.option('--sudo', is_flag=True)
 @click.pass_context
-def create_user(ctx, json, file):
+def create_user(ctx, username, email, first_name, last_name, allow_public_key, ldap_binding_user, passwordless_sudo,
```

Comment:
instead of `jccli create-user ...` command how about we setup a sub menu system so you can run:
* jccli user create ..
* jccli user get ..
* jccli user set ..
* jccli user remove ..

which would give us something similar to what's described in the [powershell command reference](https://github.com/TheJumpCloud/support/wiki/New-JCUser):
User Functions
* New JCUser
* Get JCUser
* Set JCUser
* Remove JCUser

Reply:
Okay, will do